cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -879,7 +930,11 @@ impl Node<Option<cst::VariableDef>> {
                     }
                 }
             }
-            let eref = rel_expr.to_ref_or_slot(var)?;
+            let eref = match tolerant_ast {
+                TolerantAstSetting::NotTolerant => rel_expr.to_ref_or_slot(var)?,
+                #[cfg(feature = "tolerant-ast")]
+                TolerantAstSetting::Tolerant => rel_expr.to_ref_or_slot_tolerant_ast(var)?,
```

Comment:
You don't need an `enum` for this, right? You can simply use `#[cfg(not(feature = "tolerant-ast"))]` and `#[cfg(feature = "tolerant-ast")]` to handle each case separately.

Reply:
I see, that's what I was missing. Thanks!